src/visualization/client/Turi Create Visualization/src/user_interface/src/elements/Annotate/StatusBar/index.js

```diff
@@ -19,7 +17,8 @@ class StatusBar extends Component {
         <div>
             <img src={swap}
                  className={style.SwapButton}
-                 onClick={this.swapInfiniteScroll.bind(this)}/>
+                 onClick={this.swapInfiniteScroll.bind(this)}
+	         alt="Can't Find"/>
```

Comment:
Here, the alt text should probably say something like "Switch between single image view and scrollable multi-image view".